cms/static/js/spec/views/active_video_upload_list_spec.js

```diff
@@ -51,41 +72,235 @@ define(
                 expect(this.view.onBeforeUnload()).toBeUndefined();
             });
 
-            var makeUploadUrl = function(fileName) {
+            makeUploadUrl = function(fileName) {
                 return 'http://www.example.com/test_url/' + fileName;
             };
 
-            var getSentRequests = function() {
+            getSentRequests = function() {
                 return jasmine.Ajax.requests.filter(function(request) {
                     return request.readyState > 0;
                 });
             };
 
-            describe('supported file formats', function() {
-                it('should not show unsupported file format notification for supported files', function() {
+            verifyUploadViewInfo = function(view, expectedTitle, expectedMessage) {
+                expect(view.$('.video-detail-status').text().trim()).toEqual('Upload failed');
+                expect(view.$('.more-details-action').contents().get(0).nodeValue.trim()).toEqual('Read More');
+                expect(view.$('.more-details-action .sr').text().trim()).toEqual('details about the failure');
+                view.$('a.more-details-action').click();
+                expect($('#prompt-warning-title').text().trim()).toEqual(expectedTitle);
+                expect($('#prompt-warning-description').text().trim()).toEqual(expectedMessage);
```

Comment:
Why can't we add the logic to check a11y message in this method instead of `verifyA11YMessage`?